ui-tests/src/test/java/org/openmrs/reference/page/ManageLocationAttributeTypesPage.java

```diff
@@ -0,0 +1,127 @@
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+
+public class ManageLocationAttributeTypesPage extends Page {
+
+    protected static final String PAGE_URL = "adminui/metadata/locations/manageLocationAttributeTypes.page";
+    private static final By ADD_LOCATION_ATTRIBUTE_TYPE_BUTTON = By.cssSelector("input.button");
+    private static final By ATTRIBUTE_TYPE_NAME_FIELD = By.cssSelector("#name-field");
+    private static final By ATTRIBUTE_TYPE_DESCRIPTION_FIELD = By.cssSelector("#description-field");
+    private static final By MIN_OCCURS_FIELD = By.cssSelector("#minOccurs-field");
+    private static final By MAX_OCCURS_FIELD = By.cssSelector("#maxOccurs-field");
+    private static final By DATA_TYPE_FIELD = By.cssSelector("#datatypeClassname-field");
+    private static final By DATA_TYPE_CONFIGURATION_FIELD = By.cssSelector("#datatypeConfig-field");
+    private static final By PREFERRED_HANDLER_FIELD = By.cssSelector("#preferredHandlerClassname-field");
+    private static final By HANDLER_CONFIGURATION_FILED = By.cssSelector("#handlerConfig-field");
+    private static final By SAVE_BUTTON = By.cssSelector("#save-button");
+    private static final By CANCEL_BUTTON = By.cssSelector("#locationAttributeTypeForm input.cancel");
+    private static final By EDIT_LOCATION_ATTRIBUTE_TYPE_ICON = By.cssSelector("i.icon-pencil.edit-action");
+    private static final By DELETE_LOCATION_ICON = By.cssSelector("i.icon-trash.delete-action.right");
+    private static final By CONFIRM_DELETE_LOCATION_BUTTON = By.cssSelector("#purgeLocationAttributeTypeForm button.confirm.right");
+    private static final By CONFIRM_RETIRE_LOCATION_BUTTON = By.cssSelector("#retireLocationAttributeTypeForm button.confirm.right");
+    private static final By REASON_TO_RETIRE_LOCATION_BUTTON = By.cssSelector("#retireLocationAttributeTypeForm input[type=text]");
+    private static final By CANCEL_RETIRE_LOCATION_BUTTON = By.cssSelector("#retireLocationAttributeTypeForm button.cancel");
+    private static final By CANCEL_DELETE_LOACTION_BUTTON = By.cssSelector("#purgeLocationAttributeTypeForm button.cancel");
+
+    public ManageLocationAttributeTypesPage(Page parent) {
+        super(parent);
+    }
+
+    @Override
+    public String getPageUrl() {
+        return PAGE_URL;
+    }
+    
+    public void goToAddNewLocationAttributeTypeForm() {
+        clickOn(ADD_LOCATION_ATTRIBUTE_TYPE_BUTTON);
+    }
+   
+    public void goToEditLocationAttributeTypeForm() {
+        clickOn(EDIT_LOCATION_ATTRIBUTE_TYPE_ICON);
+    }
+    
+    public void enterAttributeTypeName(String locationName) {
```

Comment:
Where are you definition locationName as a variable?

Reply:
I am thinking of giving a provision for the test implementation to set custom name of the Location within the test than setting it within the page. 

@sherrif10  any better way to work around the ideal ?

Reply:
> I am thinking of giving a provision for the test implementation to set custom name of the Location within the test than setting it within the page

Hey @kdaud , I did not get your suggestion here

Reply:
A user should be able to set Location Attribute Type name. See the UI

![UIx](https://user-images.githubusercontent.com/38831673/132324323-683f6198-7dbe-4e16-ada7-0a80972678d6.png)